agile-development/collaboration/readme.md

```diff
@@ -72,6 +72,8 @@ Knowledge sharing and bringing CSE and customer engineers together in a ‘code-
 
 ## Resources
 
+- [How to add a pairing custom field in ADO User Stories](./add-pairing-field-ado-cards.md) - adding a custom field of type _Identity_ in Azure DevOps for pairing
```

Comment:
I also recommend adding a bullet in the How to pair programming section [here](https://github.com/microsoft/code-with-engineering-playbook/tree/master/agile-development/collaboration#how-to-pair-program) of the Collaboration article as it might make it more discoverable and is a recommended practice on how to pair program effectively